LEGO1/lego/legoomni/src/common/legoanimationmanager.cpp

```diff
@@ -2826,10 +2826,10 @@ void LegoAnimationManager::FUN_10064b50(MxLong p_time)
 // FUNCTION: LEGO1 0x10064ff0
 AnimState::AnimState()
 {
-	m_unk0x0c = 0;
-	m_unk0x10 = NULL;
-	m_unk0x14 = 0;
-	m_unk0x18 = NULL;
+	m_arrayOfUnknownShortsLength = 0;
+	m_arrayOfUnknownShorts = NULL;
```

Comment:
it is preferable to keep the m_unk0x naming until we have proper names

Reply:
Do you have a suggestion? I first tried `m_unk0x18_len` and `m_unk0x18`, but that was rejected by `ncc`. We could go with `m_lengthOf0x18` if you want, but I found that even worse than the current solution.

Reply:
We shouldn't decorate the names with any kind of "hints" regarding the nature of the variable. I understand it can be tempting to do so intuitively to communicate more "work in progress" information, but too often it has turned out to be wrong and will then mislead other people who work on the codebase. The short answer is that `m_unk0x14` should remain `m_unk0x14` until we have a solid name that is supported by in-code context. Any other temporary information can be stored in. comments if you think it's helpful